imports/plugins/core/orders/client/templates/workflow/shippingTracking.js

```diff
@@ -44,7 +45,8 @@ Template.coreOrderShippingTracking.events({
   },
   "click [data-event-action=shipmentShipped]": function () {
     const template = Template.instance();
-    Meteor.call("orders/shipmentShipped", template.order, template.order.shipping[0], (err) => {
+    const shipment = getShippingInfo(template.order);
+    Meteor.call("orders/shipmentShipped", template.order, shipment, (err) => {
```

Comment:
not `(e)`, not `(err)`, but `(error)` is preferred style.

Reply:
Do we have a cascading preference or which should be used?

If we have a callback, and another method inside that callback, we can't use `error` again on the second callback since it's already defined (or at least the linter yells at you if you do). Do we do `error`, then `err`, then `e`?

Reply:
Or scope out the errors depending on what the call is doing if you have nested calls, e.g. `productError` then `orderError`?

Reply:
^^ Yes, that's a great idea.